src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -92,5 +92,32 @@ public void storeMessage(Message message) {
     }
     return users;
   }
+  /** Stores the User in Datastore. */
```

Comment:
Indentation in this block of new code is off a little. Remember each indentation level is 2 spaces.